dotcom-rendering/.gitignore

```diff
@@ -0,0 +1,48 @@
+### Node template
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+
+# Dependency directories
+node_modules
+
+# Yarn Integrity file
+.yarn-integrity
+
+# IDE files
+.idea
+.vscode/*
+!.vscode/extensions.json
+!.vscode/settings.json
+!.vscode/dcr.code-snippets
+.metals/
+
+# System files
+.DS_Store
+
+# Built files
+dist
+jsconfig.json
+target
+
+# Test coverage reports
+coverage
+
+# .git-hooks
+# Ignore auto generate husky hooks
+scripts/git-hooks/*
+!scripts/git-hooks/post-merge
+!scripts/git-hooks/install.js
+
+# Cypress result files
+/cypress/screenshots/*
+/cypress/videos/*
```

Comment:
Instead of copying the full file, could we update these lines in the root gitignore to be prefixed `/dotcom-rendering`? 

This would avoid duplication and hopefully be a bit cleaner

Reply:
Done.